stable/prometheus-operator/templates/prometheus/rules/k8s.rules.yaml

```diff
@@ -20,29 +20,36 @@ spec:
   groups:
   - name: k8s.rules
     rules:
-    - expr: sum(rate(container_cpu_usage_seconds_total{job="kubelet", image!="", container_name!=""}[5m])) by (namespace)
+    - expr: sum(rate(container_cpu_usage_seconds_total{job="kubelet", image!="", container!="POD"}[5m])) by (namespace)
```

Comment:
@bclermont `release-1.0` is used for Kubernetes <1.14 specifically because of breaking changes that will be introduced from 1.15 onwards for metrics.

Reply:
@vsliouniaev you mean `release-0.1` ?

Reply:
Yes, you're right!